default/scripting/species/common/industry.py

```diff
@@ -1,11 +1,12 @@
+from species.common.multiplier import GOOD_MULTIPLIER, GREAT_MULTIPLIER
```

Comment:
Ruff considered species as 3d party and common as 1 party so it reformat imports and put a space between groups. 

To fix it we need to add `default/scripting/` to the `src` section in pyproject.toml

Reply:
Changed it